### ISSUE TITLE
setArrowDrawable not show as expected arrow.

### ISSUE THREAD

**Issue by popeyee27** [skydoves/Balloon#461]
**Please complete the following information:**
- Library Version: 1.5.3
- Affected Device(s) : Oppo A92 (Android 11), Emulator Pixel XL (Android 13)
 
**Describe the Bug:**
I try to custom tooltip arrow with 
`.setArrowDrawable(ContextCompat.getDrawable(context, {drawable))` in my custom layout.

**The drawable with size 16x8 **
<img width="622" alt="Screenshot 2566-06-14 at 15 09 18" src="https://github.com/skydoves/Balloon/assets/8433930/4b9d6d87-f954-4abb-a36d-cbbbac4c8301">

However the arrow not show as expected.  With `setArrowDrawable()`, It look like I only little round my arrow.

**with setArrowDrawable**
<img width="735" alt="Screenshot 2566-06-14 at 15 21 57" src="https://github.com/skydoves/Balloon/assets/8433930/151f39df-e95c-45da-b4ff-06f614f53aa0">

**default**
<img width="737" alt="Screenshot 2566-06-14 at 15 22 49" src="https://github.com/skydoves/Balloon/assets/8433930/575a8ce6-069e-4edc-b45e-ce3c56034ba9">

**Expected Behavior:**
I has expected the arrow to like this no matter size of the balloon.
<img width="812" alt="Screenshot 2566-06-14 at 15 17 27" src="https://github.com/skydoves/Balloon/assets/8433930/4cf33ec2-5272-47b7-98de-b01299fb8110">

Should I fixed my arrow vector or set another thing? I already set `setArrowPositionRules(ALIGN_ANCHOR)`, `setArrowPosition(0.5f)`,`setArrowOrientation(ArrowOrientation.TOP)` and `setArrowSize(12)` before ` .setArrowDrawable()`

**Comment by skydoves** [comment]
The arrow must be decided by the size of your arrow. Would you able to increase the arrow size with `setArrowSize(12)` method?

**Comment by popeyee27** [comment]
yes, no matter I call `setArrowSize()` before or after `.setArrowDrawable()`. I able to increase a size of arrow but it look like not my drawable arrow. 

``` .setArrowDrawable(ContextCompat.getDrawable(context, R.drawable.lmds_v3_ic_tooptip_arrow))
      .setArrowSize(8)```

<img width="703" alt="Screenshot 2566-06-14 at 17 20 49" src="https://github.com/skydoves/Balloon/assets/8433930/2c252984-7653-4a1c-b5c5-d27e84dab123">

```
.setArrowSize(8)
.setArrowDrawable(ContextCompat.getDrawable(context, R.drawable.lmds_v3_ic_tooptip_arrow))
```
<img width="723" alt="Screenshot 2566-06-14 at 17 22 30" src="https://github.com/skydoves/Balloon/assets/8433930/a5a08870-ee82-4b30-80a5-45eba53d2f0f">

**only `.setArrowDrawable(ContextCompat.getDrawable(context, R.drawable.lmds_v3_ic_tooptip_arrow))`**
<img width="719" alt="Screenshot 2566-06-14 at 17 23 38" src="https://github.com/skydoves/Balloon/assets/8433930/228a5c3c-43fd-48cf-b696-e24be3d7bab0">


```
.setArrowSize(20)
.setArrowDrawable(ContextCompat.getDrawable(context, R.drawable.lmds_v3_ic_tooptip_arrow))
```
<img width="725" alt="Screenshot 2566-06-14 at 17 24 24" src="https://github.com/skydoves/Balloon/assets/8433930/97114943-d6ef-4c65-9f6c-8b1da8a5ce02">

**Comment by alexanderfilipzik** [comment]
ping, i think it's because the library will render it as a square, we have the same issue here

**Comment by skydoves** [comment]
They guys thanks for raising this issue. I will take a look at this, but I believe it will take some time to resolve this issue since the squared-arrow size already has been used overall project. In the meantime, you can adjust your arrow image file as square or include some transparent padding on your image file itself following your preferences.

**Comment by kikin81** [comment]
I'm having the same issue. My design calls for an arrow with 28.dp width and 12.dp height. 
We tried doing the drawable trick of adding more padding top but this results on the Balloon popover being too far away from the target as you can see in this screenshot.

Hope we can have custom width and height!

![nubbin outline](https://github.com/skydoves/Balloon/assets/834204/8387f060-f8ec-469d-8911-7b6be7899ece)

**Comment by codercoderme** [comment]
Facing the same issue.     while I'm trying to add a border （include arrow）

**Comment by ziracook-qz** [comment]
+1 to this issue
The padding trick doesn't seem to work with custom drawables
I had to workaround by using an icon with padding built in which isn't super ideal
